test/factories/code_schools.rb

```diff
@@ -0,0 +1,12 @@
+FactoryGirl.define do
+  factory :code_school do
+    name "CoderSchool"
```

Comment:
Perhaps leverage the [Faker gem](https://github.com/OperationCode/operationcode_backend/blob/master/test/factories/users.rb) to create unique `name`s, `url`s, `addresses`(in the other `location` factory). 

Here are [some examples](https://github.com/OperationCode/operationcode_backend/blob/master/test/factories/users.rb#L4-L12) of the gem being used.

Reply:
Can you update these with `Faker`.